### PR TITLE
Migrate identifier literal test cases

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java [hunk 1]
```diff
@@ -189,26 +186,30 @@ public void testIdentifierLiteralInStructName() {
...
-    @Test(description = "Test error message when trying to access undefined global var with identifier literal",
-            expectedExceptions = {SemanticException.class},
-            expectedExceptionsMessageRegExp = "identifier-literal-undefined-variable.bal:5: undefined " +
-                    "symbol 'global v \" ar'")
+    @Test(description = "Test error message when trying to access undefined global var with identifier literal")
     public void testTryToAccessUndefinedGlobalVarWithIdentifierLiteral() {
-        BTestUtils.getProgramFile("lang/identifierliteral/identifier-literal-undefined-variable.bal");
+        CompileResult resultNeg = BTestUtils
+                .compile("test-src/expressions/identifierliteral/identifier-literal-undefined-variable.bal");
+        Assert.assertEquals(resultNeg.getErrorCount(), 1);
```
Comment (anupama-pathirage) [inline modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java]: Shall we rename this to *-negative.bal

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java [hunk 2]
```diff
@@ -189,26 +186,30 @@ public void testIdentifierLiteralInStructName() {
...
 
-    @Test(description = "Test wrong character in identifier literal",
-            expectedExceptions = {ParserException.class},
-            expectedExceptionsMessageRegExp = "identifier-literal-wrong-character.bal:3:21: missing ';' before 'var'")
+    @Test(description = "Test wrong character in identifier literal")
     public void testIdentifierLiteralWithWrongCharacter() {
-        BTestUtils.getProgramFile("lang/identifierliteral/identifier-literal-wrong-character.bal");
+        CompileResult resultNeg = BTestUtils
+                .compile("test-src/expressions/identifierliteral/identifier-literal-wrong-character.bal");
+        Assert.assertEquals(resultNeg.getErrorCount(), 3);
```
Comment (anupama-pathirage) [inline modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java]: Shall we rename this to *-negative.bal

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java (hunk 1 above)
Comment (chamil321) [reply modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java]: done

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java (hunk 2 above)
Comment (chamil321) [reply modules/ballerina-test/src/test/java/org/ballerinalang/test/expressions/identifierliteral/IdentifierLiteralTest.java]: done

**Pull request merged**